../output.c:18:10: fatal error: wlr/backend/x11.h: No such file or directory

It's weird that meson didn't whine about a missing library.

Debian GNU/Linux buster/sid
wlroots and cage master

I built wlroots normally (as suggested on their github page).

```
$ meson build
The Meson build system
Version: 0.49.2
Source dir: /home/bbigras/cage
Build dir: /home/bbigras/cage/build
Build type: native build
Project name: cage
Project version: 0.0.1
Native C compiler: cc (gcc 8.2.0 "cc (Debian 8.2.0-20) 8.2.0")
Build machine cpu family: x86
Build machine cpu: i686
Found pkg-config: /usr/bin/pkg-config (0.29)
Dependency wlroots found: YES 0.3.0
Dependency wayland-protocols found: YES 1.17
Dependency wayland-server found: YES 1.16.0
Dependency pixman-1 found: YES 0.36.0
Dependency xkbcommon found: YES 0.8.2
Library m found: YES
Program wayland-scanner found: YES (/usr/bin/wayland-scanner)
Configuring config.h using configuration
Message:
Cage 0.0.1

    xwayland: 0

Build targets in project: 1
Option werror is: True [default: true]
Found ninja-1.8.2 at /usr/bin/ninja
```

```
$ LANG=c ninja -C build
ninja: Entering directory `build'
[1/3] Compiling C object 'cage@exe/output.c.o'.
FAILED: cage@exe/output.c.o
cc -Icage@exe -I. -I.. -I/usr/local/include -I/usr/include/libdrm -I/usr/include/pixman-1 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -Werror -std=c11 -g -DWLR_USE_UNSTABLE -Wall -Werror -Wundef -Wno-unused-parameter -DDEBUG  -MD -MQ 'cage@exe/output.c.o' -MF 'cage@exe/output.c.o.d' -o 'cage@exe/output.c.o' -c ../output.c
../output.c:18:10: fatal error: wlr/backend/x11.h: No such file or directory
 #include <wlr/backend/x11.h>
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
[2/3] Compiling C object 'cage@exe/xdg_shell.c.o'.
ninja: build stopped: subcommand failed.
```

--- a/output.c
+++ b/output.c
@@ -9,13 +9,16 @@
 #define _POSIX_C_SOURCE 200112L
 
 #include "config.h"
+#include <wlr/config.h>
 
 #include <stdlib.h>
 #include <unistd.h>
 #include <wayland-server.h>
 #include <wlr/backend.h>
 #include <wlr/backend/wayland.h>
+#if WLR_HAS_X11_BACKEND
 #include <wlr/backend/x11.h>
+#endif
 #include <wlr/render/wlr_renderer.h>
 #include <wlr/types/wlr_data_device.h>
 #include <wlr/types/wlr_matrix.h>
@@ -424,7 +427,9 @@ output_set_window_title(struct cg_output *output, const char *title)
 
 	if (wlr_output_is_wl(wlr_output)) {
 		wlr_wl_output_set_title(wlr_output, title);
+#if WLR_HAS_X11_BACKEND
 	} else if (wlr_output_is_x11(wlr_output)) {
 		wlr_x11_output_set_title(wlr_output, title);
+#endif
 	}
 }

